src/group/managed_group/mod.rs

```diff
@@ -530,7 +530,15 @@ impl<'a> ManagedGroup<'a> {
                                 )));
                             }
                             _ => {
-                                panic!("apply_commit_error did not return an ApplyCommitError.");
+                                let error_string =
+                                    "apply_commit() did not return an ApplyCommitError."
+                                        .to_string();
+                                events.push(GroupEvent::Error(ErrorEvent::new(
+                                    ManagedGroupError::LibraryError(ErrorString::from(
+                                        error_string,
+                                    )),
+                                )));
+                                panic!();
```

Comment:
The commit message says "avoid a panic", but this is still panicking. Is that intentional?

Reply:
With this panic it doesn't make sense to have a new event. So I guess this should go...